docker.sh

```diff
@@ -32,9 +32,7 @@ echo ""
 if [ "${DEPEND}" == "build" ]; then
     echo "# To just pull latest/develop version (default) run"
     echo "# DEPEND=pull $0"
-    ${DEBUG} docker build \
-        --tag pecan/depends:latest \
-        --file docker/base/Dockerfile.depends .
+    ${DEBUG} docker build --tag pecan/depends:latest docker/depends
```

Comment:
Can you make the `latest` here (and other places) user-configurable via `DEPENDS_VERSION=${DEPENDS_VERSION:-latest}` or something?

Reply:
Using IMAGE_VERSION for the version. Can now use `DEPEND=pull IMAGE_VERSION=pr ./docker.sh` to create images that are tagged with pr.  The DEPEND=pull is needed to make sure we have a depends image that is tagged correctly (you can also use DEPEND=build but that will take a while).